inscriptions/ISic0889.xml

```diff
@@ -170,24 +175,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140370">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    𐆠 Ἀθανασίου. ΧΜΓ.
+                    <lb n="1"/>Ἀθανασίου. ΧΜΓ.
                     <lb n="2"/>
-                    <surplus>²per medium litteris rubreis:</surplus>²<lb n="3"/>Ἀλέξανδρος.
+                    <lb n="3"/>Ἀλέξανδρος
                     <lb n="4"/>
-                    <surplus>²infra:</surplus>²<lb n="5"/>α ☧ ω ☧ϲ <surplus>²⁶Χρ(ιστό)ς</surplus>²⁶
-                    <surplus>☧</surplus>
-                    
-                    ☧ϲ
-                    <lb n="6"/>
+                    <lb n="5"/>α ☧ ω ☧ϲ ☧ϲ
```

Comment:
use of <g> on the chi-rho? Note the second is Χρ(ιστό)ς, the third however less clear.